lib/Account.php

```diff
@@ -144,10 +149,44 @@ public function getImapConnection() {
 						))];
 				}
 			}
-			$this->client = new \Horde_Imap_Client_Socket($params);
-			$this->client->login();
+			$this->imapClient = new \Horde_Imap_Client_Socket($params);
+			$this->imapClient->login();
 		}
-		return $this->client;
+		return $this->imapClient;
+	}
+
+	/**
+	 * @return ManageSieve
+	 * @throws ManageSieve\Exception
+	 */
+	public function getSieveConnection()
```

Comment:
`getImapConnection` is using an outdated coding style and we're int process of refactoring our code to get rid of the method. Please see the `ImapClientFactory` and create something similar for Sieve.